src/common/drydep.f90

```diff
@@ -91,12 +92,14 @@ pure subroutine drydep_precompute(surface_pressure, t2m, yflux, xflux, z0, &
     roa, ustar, monin_obukhov_length, raero, vs, rs, date)
   use iso_fortran_env, only: real64, int8
   use datetime, only: datetime_t
-  real, intent(in) :: surface_pressure(:,:) !> [hPa]
-  real, intent(in) :: t2m(:,:)
+  real, intent(in) :: surface_pressure(:,:) !> [Pa]
+  real, intent(in) :: t2m(:,:) !> [K]
   real, intent(in) :: yflux(:,:), xflux(:,:)
   real, intent(in) :: z0(:,:), hflux(:,:)
   real, intent(in) :: leaf_area_index(:,:)
+  !> Diameter in m
```

Comment:
units of `xflux, yflux, hflux, z0`?

Why is `leaf_area_index`  not integer if it is a index?

Reply:
What is classnr?

Reply:
Added units to xflux and friends. `leaf_area_index` is unitless but not an integer: https://en.wikipedia.org/wiki/Leaf_area_index